### PR TITLE
kubeadm reference for 1.25

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_generate-csr.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Generate keys and certificate signing requests
 
 Generates keys and certificate signing requests (CSRs) for all the certificates required to run the control plane. This command also generates partial kubeconfig files with private key data in the  "users &gt; user &gt; client-key-data" field, and for each kubeconfig file an accompanying ".csr" file is created.
 
-This command is designed for use in [Kubeadm External CA Mode](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#external-ca-mode). It generates CSRs which you can then submit to your external certificate authority for signing.
+This command is designed for use in [Kubeadm External CA Mode](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#external-ca-mode). It generates CSRs which you can then submit to your external certificate authority for signing.
```
Comment (jihoon-seo) [inline content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_generate-csr.md:20]: Would it be possible for us to remove this hardcoded site domain name?

```suggestion
This command is designed for use in [Kubeadm External CA Mode](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#external-ca-mode). It generates CSRs which you can then submit to your external certificate authority for signing.
```

**Review by jihoon-seo — COMMENTED** [review]
@tengqm PTAL!

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_generate-csr.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_generate-csr.md:20]: This is generated out of upstream source code. We may have to propose a fix there if desired.

**Comment by reylejano** [comment]
A few previews:
- https://deploy-preview-36207--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm/
- https://deploy-preview-36207--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs/
- https://deploy-preview-36207--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_certificate-key/
- https://deploy-preview-36207--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_check-expiration/
- https://deploy-preview-36207--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_generate-csr/

/lgtm

**Pull request merged**